src/brevitas/graph/gpxq.py

```diff
@@ -110,6 +113,8 @@ def _is_module_supported(self, module):
             return False
 
     def __enter__(self):
+        # Disable quantization selectively
+        super().__enter__()
```

Comment:
Ah ignore the previous comments, I guess with composibility we couldn't do this right?